server/index.js

```diff
@@ -12,27 +12,13 @@ app.use(express.json());
 
 const PORT = process.env.PORT || 8080;
 
-// For testing purposes
-// app.post('/test-auth', authMiddleware, (req, res) => {
-//   const user = req.user;
-
-//   const userInfo = {
-//       id: user.id,
-//       username: user.username,
-//       email: user.email,
-//       role: user.role,
-//   };
-
-//   res.json(userInfo);
-// });
-
 // Sync Sequelize models
 db.sequelize.sync().then(async () => {
   // Use { force: true } cautiously as it will drop existing tables
   console.log("Database synced");
 
   // import the challenges into the db. Comment out after first run
-  // await importChallenges();
+  await importChallenges();
```

Comment:
intentionally leaving this uncommented?

Reply:
yes. I will delete the the `comment out` part